aten/src/TH/generic/THTensorMath.c

```diff
@@ -1070,7 +1070,7 @@ void THTensor_(remainder)(THTensor *r_, THTensor *t, real value)
 #else
       // There is no NAN for integers
       rp[i] = tp[i] % value;
-      if (rp[i] * value < 0)
+      if ((rp[i] < 0) != (value < 0))
```

Comment:
nit: It would be nicer to have this as a macro (and/or add comments about why this is necessary). Something like:
```
#define HAS_DIFFERENT_SIGN(x, y)

// undef this when it's not necessary
#undef HAS_DIFFERENT_SIGN
````

Reply:
Or better a `static inline` function 😊